docs/pages/classes/Components

```diff
@@ -0,0 +1,209 @@
+---
```

Comment:
@TimPurdum All of these files (in `classes`) are auto-generated, correct? Do we really want to check them all in?

Reply:
They are auto-generated, but only by building `dymaptic.GeoBlazor.Core`. So we'd have to customize the GH pages build to generate them. I haven't gotten that far in my knowledge.

Reply:
I see, okay.